__tests__/dao/sqlUtils.test.ts

```diff
@@ -0,0 +1,136 @@
+import { sqlUtils } from '../../src/dao'
+
+describe('stringifyStatement', () => {
+  it('Should return the array formatted with separator', () => {
+    expect(sqlUtils.stringifyStatement(['banana', 'uva'], ', ')).toEqual('banana, uva')
+  })
+  it('Should return the value formatted', () => {
+    expect(sqlUtils.stringifyStatement('uva')).toEqual('uva')
+  })
+})
+
+describe('normalizeColumnName', () => {
+  it('Should return the column name', () => {
+    expect(sqlUtils.normalizeColumnName('cesta.banana', 'cesta')).toEqual('`cesta`.`banana`')
+  })
+  it('Should return the column name', () => {
+    expect(sqlUtils.normalizeColumnName('uva', 'cesta')).toEqual('`cesta`.`uva`')
```

Comment:
if you just pass normalizeColumnName('banana')?

Reply:
i need to pass at least 2 args

Reply:
Why?